src/indent.cpp

```diff
@@ -430,7 +427,7 @@ static void indent_pse_pop(parse_frame_t &frm, chunk_t *pc)
    {
       if (pc != NULL)
       {
-         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %d, level=%zu/%zu\n",
+         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %zu, level=%zu/%zu\n",
```

Comment:
The format "z" is for size_t.  The more portable method is to use the PRIuS macro as found in inttypes.h.

"...starting on line " PRIuS ", level..."